src/transformer/__tests__/__snapshots__/decode.test.ts.snap

```diff
@@ -5744,6 +5714,47 @@ Object {
       ],
       "type": "section",
     },
+    Object {
+      "attrs": Object {
+        "id": "undefined:9158450B-F1AF-44B7-B884-4EDC4AA27107",
```

Comment:
Is that ok that is it "undefined"?

Reply:
Oh, since there is no ObjectType associated with this node the id would look like this, it won't break anmything, but yeah it doesn't look good

Reply:
would it be better to remove the whole attr section (this object is just a placeholder, it will not be represented in the modelMap)